source/manual/get-started.html.md

```diff
@@ -161,11 +161,17 @@ Once your VM is running, you can SSH into it with:
 
 ### Set your Git username and email
 
-You can assign your name and email to commits on the VM:
+You can associate your GitHub account with commits on the VM by setting your git global configs on the dev machine.
```

Comment:
I don't think this wording is 100% accurate.  You're only setting an arbitrary name/email, which may or may not be linked to a GitHub account.

Reply:
This is true 👍 Will fix